src/v3/verify.ts

```diff
@@ -15,12 +15,10 @@ export const verify = <T extends OpenAttestationVerifiableCredential>(
   const { proof, ...documentWithoutProof } = document;
   const decodedSalts = decodeSalt(document.proof.salts);
 
-  // Count number of hashes: count(visible) + count(obfuscated) should match count(decodedSalts)
+  // Count number of hashes: count(visible) should match count(decodedSalts)
   const countVisible = salt(documentWithoutProof).length;
-  const countObfuscated = document.proof.privacy.obfuscated.length;
-  const count = countVisible + countObfuscated;
 
-  if (count !== decodedSalts.length) return false;
+  if (countVisible !== decodedSalts.length) return false;
```

Comment:
I originally thought that obfuscating a key would result in (a) the data removed, but (b) the salt is still there. It was wrong. So this check has been edited to `count(visible data) should match count(decoded salts)`.

I did not `+ countObfuscated` to both sides since it would have cancelled out.

Reply:
:thinking: 

I suggest to add the reason of `Count number of hashes: count(visible) should match count(decodedSalts)` (to make sure there is no added / removed values), otherwise I think it's hard to understand why we are doing this check